Gordon360/Controllers/NewsController.cs

```diff
@@ -198,5 +199,26 @@ public ActionResult<StudentNewsViewModel> EditPosting(int newsID, [FromBody] Stu
             var result = _newsService.EditPosting(newsID, studentNewsEdit);
             return Ok(result);
         }
+
+        /// <summary>
+        ///  Approve a news posting in the database
+        /// </summary>
+        /// <param name="newsID">The id of the news item to approve</param>
+        /// <returns>The approved news item</returns>
+        /// <remarks>The news item must not be expired and must be unapproved</remarks>
+        [HttpPut]
+        [Route("approvalStatus")]
+        // only SNAdmin is authorized to use this route
+        // Can't use SYB UPDATE operation, because the author is not authorized to approve the post
+        public ActionResult<StudentNewsViewModel> ApprovePosting([FromBody] int newsID)
+        {
+            if (user_groups.Contains(AuthGroup.SiteAdmin) || user_groups.Contains(AuthGroup.NewsAdmin))
+            {
+                var result = _newsService.ApprovePosting(newsID);
```

Comment:
Then here, you should probably have an 
```
if newStatusAccepted
...
else
// unaccept news item
```

Either that or just change the service method to take in a Boolean option as well and handle the case for unapproving the student news.